src/api/authApi.ts

```diff
@@ -50,4 +50,20 @@ export const authApi = {
       });
     return response;
   },
+
+  registrationClient: async (data: {} ) => {
+    const response = await authInstance
+      .post(`api/client/registration`, data)
+      .then((res: AxiosResponse<ILoginResponse>) => {
+        console.log(`POST [api/client/registration/${data}] response received successfully`);
+        console.log(res);
+        return res.data;
+      })
+      .catch((error: AxiosError<ILoginResponse>) => {
+        // place to handle errors and rise custom errors
+        console.log(`POST [api/client/registration/${data}] error message: ${error.message}`);
+        throw error.message;
+      });
```

Comment:
I find it generaly bad idea to combine awaits and then syntax. I'd rather stick only to async/await

Reply:
I understood
but I don't know how to write it correctly, but I have to replace the then => awaite